  INFO root [2018-10-23 00:00:00,477] - Components::findComponents: find logger(1)...
  INFO root [2018-10-23 00:00:00,581] - Components::findComponents: found logger, addr:192.168.0.107:57710
  INFO root [2018-10-23 00:00:00,581] - Components::addComponent: reset orderLog, uid=32104!
  INFO root [2018-10-23 00:00:00,582] - Components::addComponent[logger], uid=32104, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-23 00:00:00,584] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-10-23 00:00:00,692] - Components::findComponents: found dbmgr, addr:192.168.0.107:57703
  INFO root [2018-10-23 00:00:00,693] - Components::addComponent: reset orderLog, uid=32104!
  INFO root [2018-10-23 00:00:00,693] - Components::addComponent[dbmgr], uid=32104, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-23 00:00:00,694] - Components::findComponents: found dbmgr, addr:192.168.0.107:57703
  WARN root [2018-10-23 00:00:01,694] - Forwarding logs to logger[192.168.0.107:57710/0/0/0]...
 ERROR root [2018-10-23 00:07:16,846] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-23 00:07:16,849] - The following logs sent to logger failed:
  WARN root [2018-10-23 00:07:16,850] - ==> [2018-10-23 00:07:16 845] TCPPacketReceiver::processPendingEvents(192.168.0.107:57710): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-23 00:07:16,850] - ==> [2018-10-23 00:07:16 846] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-10-23 00:07:16,897] - TCPPacketReceiver::processPendingEvents(192.168.0.107:57703): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-23 00:07:16,897] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-10-23 00:07:16,898] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-10-23 00:07:16,900] - onLoginAppShutDown()
  INFO root [2018-10-23 00:07:16,900] - Shutdowner::onShutdown: shutting down
  INFO root [2018-10-23 00:07:16,900] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-10-23 00:07:16,900] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-23 00:07:16,901] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-23 00:07:16,911] - Script::uninstall(): is successfully!
 DEBUG root [2018-10-23 00:07:16,912] - ThreadPool::destroy(): starting size 2.
  INFO root [2018-10-23 00:07:18,046] - 
  INFO root [2018-10-23 00:07:18,047] - -----------------------------------------------------------------------------------------


  INFO root [2018-10-23 00:07:18,048] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-10-23 00:07:18,048] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-10-23 00:07:18,048] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-10-23 00:07:18,052] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20013 )
  INFO root [2018-10-23 00:07:18,052] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20013, SOMAXCONN=128.
  INFO root [2018-10-23 00:07:18,053] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:57835 )
  INFO root [2018-10-23 00:07:18,054] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:57835, SOMAXCONN=128.
  INFO root [2018-10-23 00:07:18,054] - server-configs:
  INFO root [2018-10-23 00:07:18,054] - 	internalAddr : 192.168.0.107:57835
  INFO root [2018-10-23 00:07:18,054] - 	externalAddr : 192.168.0.107:20013
  INFO root [2018-10-23 00:07:18,054] - 	componentID : 9000
  INFO root [2018-10-23 00:07:19,748] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:53:07 Apr 29 2018. AppID: 9000. UID: 796. PID: 8264 ----
  INFO root [2018-10-23 00:07:19,749] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-10-23 00:07:19,749] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-10-23 00:07:19,749] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-10-23 00:07:19,872] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-23 00:07:19,873] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-10-23 00:07:19,873] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-23 00:07:19,876] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-23 00:07:19,876] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-10-23 00:07:19,876] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-23 00:07:19,877] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-23 00:07:19,906] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-23 00:07:19,934] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-10-23 00:07:19,935] - Script::install(): KBExtra is successfully!
S_INFO root [2018-10-23 00:07:19,936] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-10-23 00:07:19,936] - TelnetServer server is running on port 31000
  INFO root [2018-10-23 00:07:20,225] - ---- loginapp is running ----
 DEBUG root [2018-10-23 00:07:20,226] - Components::process(): Request for the process of identity...
  WARN root [2018-10-23 00:07:20,536] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20086, errorno=10048
  INFO root [2018-10-23 00:07:27,678] - Components::findComponents: find logger(1)...
  INFO root [2018-10-23 00:07:27,771] - Components::findComponents: found logger, addr:192.168.0.107:57834
  INFO root [2018-10-23 00:07:27,771] - Components::addComponent: reset orderLog, uid=796!
  INFO root [2018-10-23 00:07:27,771] - Components::addComponent[logger], uid=796, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-23 00:07:27,772] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-10-23 00:07:27,873] - Components::findComponents: found dbmgr, addr:192.168.0.107:57830
  INFO root [2018-10-23 00:07:27,873] - Components::addComponent: reset orderLog, uid=796!
  INFO root [2018-10-23 00:07:27,873] - Components::addComponent[dbmgr], uid=796, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-23 00:07:27,873] - Components::findComponents: found dbmgr, addr:192.168.0.107:57830
  WARN root [2018-10-23 00:07:28,873] - Forwarding logs to logger[192.168.0.107:57834/0/0/0]...
 ERROR root [2018-10-23 00:10:19,741] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-23 00:10:19,742] - The following logs sent to logger failed:
  WARN root [2018-10-23 00:10:19,742] - ==> [2018-10-23 00:10:19 740] TCPPacketReceiver::processPendingEvents(192.168.0.107:57834): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-23 00:10:19,742] - ==> [2018-10-23 00:10:19 741] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-10-23 00:10:19,792] - TCPPacketReceiver::processPendingEvents(192.168.0.107:57830): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-23 00:10:19,792] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-10-23 00:10:19,795] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-10-23 00:10:19,798] - onLoginAppShutDown()
  INFO root [2018-10-23 00:10:19,798] - Shutdowner::onShutdown: shutting down
  INFO root [2018-10-23 00:10:19,798] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-10-23 00:10:19,798] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-23 00:10:19,798] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-10-23 00:10:19,798] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-23 00:10:19,798] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-23 00:10:19,798] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-10-23 00:10:19,798] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-23 00:10:19,799] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-23 00:10:19,799] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-23 00:10:19,807] - Script::uninstall(): is successfully!
 DEBUG root [2018-10-23 00:10:19,808] - ThreadPool::destroy(): starting size 2.
  INFO root [2018-10-23 00:10:20,936] - 
  INFO root [2018-10-23 00:10:20,936] - -----------------------------------------------------------------------------------------


  INFO root [2018-10-23 00:10:20,937] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-10-23 00:10:20,937] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-10-23 00:10:20,937] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-10-23 00:10:20,941] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20013 )
  INFO root [2018-10-23 00:10:20,941] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20013, SOMAXCONN=128.
  INFO root [2018-10-23 00:10:20,942] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:57907 )
  INFO root [2018-10-23 00:10:20,942] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:57907, SOMAXCONN=128.
  INFO root [2018-10-23 00:10:20,942] - server-configs:
  INFO root [2018-10-23 00:10:20,942] - 	internalAddr : 192.168.0.107:57907
  INFO root [2018-10-23 00:10:20,943] - 	externalAddr : 192.168.0.107:20013
  INFO root [2018-10-23 00:10:20,943] - 	externalCustomAddr : 127.0.0.1
  INFO root [2018-10-23 00:10:20,943] - 	componentID : 9000
  INFO root [2018-10-23 00:10:22,647] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:53:07 Apr 29 2018. AppID: 9000. UID: 1394. PID: 7960 ----
  INFO root [2018-10-23 00:10:22,648] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-10-23 00:10:22,648] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-10-23 00:10:22,648] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-10-23 00:10:22,774] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-23 00:10:22,775] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-10-23 00:10:22,775] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-23 00:10:22,778] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-23 00:10:22,778] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-10-23 00:10:22,778] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-23 00:10:22,779] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-23 00:10:22,808] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-23 00:10:22,847] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-10-23 00:10:22,847] - Script::install(): KBExtra is successfully!
S_INFO root [2018-10-23 00:10:22,850] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-10-23 00:10:22,850] - TelnetServer server is running on port 31000
  INFO root [2018-10-23 00:10:23,167] - ---- loginapp is running ----
 DEBUG root [2018-10-23 00:10:23,167] - Components::process(): Request for the process of identity...
  INFO root [2018-10-23 00:10:30,270] - Components::findComponents: find logger(1)...
  INFO root [2018-10-23 00:10:30,362] - Components::findComponents: found logger, addr:192.168.0.107:57906
  INFO root [2018-10-23 00:10:30,362] - Components::addComponent: reset orderLog, uid=1394!
  INFO root [2018-10-23 00:10:30,362] - Components::addComponent[logger], uid=1394, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-23 00:10:30,364] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-10-23 00:10:30,466] - Components::findComponents: found dbmgr, addr:192.168.0.107:57904
  INFO root [2018-10-23 00:10:30,467] - Components::addComponent: reset orderLog, uid=1394!
  INFO root [2018-10-23 00:10:30,467] - Components::addComponent[dbmgr], uid=1394, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-23 00:10:30,467] - Components::findComponents: found dbmgr, addr:192.168.0.107:57904
  WARN root [2018-10-23 00:10:31,468] - Forwarding logs to logger[192.168.0.107:57906/0/0/0]...
